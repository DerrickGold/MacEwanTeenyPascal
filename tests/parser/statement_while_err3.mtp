begin

	(* while statement: missing while *)
	10 do
		sample := sample + 1
end
.