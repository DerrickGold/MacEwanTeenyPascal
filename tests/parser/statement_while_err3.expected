Syntax Error: line 4:
Unexpected token (TOK_NUM, '10'), likely missing a(n)  
  'TOK_SEMICOLON' or
Possibly due to malformed or missing:
  Non-terminal: 'If Statement' or
  Non-terminal: 'While Statement' or
  Non-terminal: 'Write Statement' or
  Non-terminal: 'Read Statement' or
  Non-terminal: 'Case'
